parser/prism/Translator.cc

```diff
@@ -1302,6 +1303,15 @@ unique_ptr<parser::Node> Translator::translate(pm_node_t *node, bool preserveCon
                 supportedBlock = true;
             }
 
+            if (hasFwdArgs) { // Desugar a call like `foo(...)` so it has a block argument like `foo(..., &b)`.
```

Comment:
Equivalent to:

https://github.com/Sorbet/sorbet/blob/fb4fe38c4828cf9c054768413256b4982efe7d05/ast/desugar/Desugar.cc#L769-L772